analyses/transcriptomic-dimension-reduction/02-transcriptomic-analysis-plotting.Rmd

```diff
@@ -0,0 +1,650 @@
+  )))
+rsem_pca_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "rsem_pca_scores_aligned.tsv"
+  )))
+poly_rsem_tsne_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "poly_rsem_tsne_scores_aligned.tsv"
+  )))
+stranded_rsem_tsne_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "stranded_rsem_tsne_scores_aligned.tsv"
+  )))
+rsem_tsne_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "rsem_tsne_scores_aligned.tsv"
+  )))
+poly_rsem_umap_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "poly_rsem_umap_scores_aligned.tsv"
+  )))
+stranded_rsem_umap_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "stranded_rsem_umap_scores_aligned.tsv"
+  )))
+rsem_umap_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "rsem_umap_scores_aligned.tsv"
+  )))
+
+### Kallisto -------------------------------------------------------------------
+poly_kallisto_pca_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "poly_kallisto_pca_scores_aligned.tsv"
+  )))
+stranded_kallisto_pca_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "stranded_kallisto_pca_scores_aligned.tsv"
+  )))
+kallisto_pca_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "kallisto_pca_scores_aligned.tsv"
+  )))
+poly_kallisto_tsne_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "poly_kallisto_tsne_scores_aligned.tsv"
+  )))
+stranded_kallisto_tsne_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "stranded_kallisto_tsne_scores_aligned.tsv"
+  )))
+kallisto_tsne_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "kallisto_tsne_scores_aligned.tsv"
+  )))
+poly_kallisto_umap_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "poly_kallisto_umap_scores_aligned.tsv"
+  )))
+stranded_kallisto_umap_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "stranded_kallisto_umap_scores_aligned.tsv"
+  )))
+kallisto_umap_aligned_df <-
+  (readr::read_tsv(file.path(
+    results_dir, "kallisto_umap_scores_aligned.tsv"
+  )))
+```
+
+
+# Plot RSEM
+
+## Combined RSEM Plots Grid
+The grid of plots below shows the dimension reductions scores for the RSEM 
+expression data, colored by broad histology and selection strategy.
+These plots seem to suggest the presence of batch effects, which is likely due 
+to the need for normalization across the two batches of RNA-Seq preparation 
+(poly-A selection versus stranded(ribosomal depletion)).
+This can be further determined using the `RNA_library` column of the
+metadata to attempt to correct batch effects as suggested by the selection
+strategy grid of plots.
+
+```{r, fig.width = 10, fig.height = 18}
+# Run the `plot_dimension_reduction` function to plot each set of dimension
+# reduction scores using `broad_histology` to color points
+rsem_pca_plot_histology <-
+  plot_dimension_reduction(rsem_pca_aligned_df, "broad_histology", 1, 5)
```

Comment:
I know I said I was curious about other PCs, but lets just stick to 1 & 2 for the main plots here (now that I've seen more).